lib/treeherder.py

```diff
@@ -6,14 +6,14 @@
 from thclient import TreeherderClient
 
 
-def get_revision_hash(host, project, revision):
+def get_revision_hash(server_url, project, revision):
```

Comment:
We may have to return to use taskcluster soon again. So I'm not in favor of removing this code now. We can evaluate that later.